lib/oban/testing.ex

```diff
@@ -116,7 +116,13 @@ defmodule Oban.Testing do
   import Ecto.Query, only: [limit: 2, order_by: 2, select: 2, where: 2, where: 3]
 
   alias Ecto.Changeset
-  alias Oban.{Job, Repo, Worker}
+
+  alias Oban.{
+    Job,
+    Queue.Executor,
+    Repo,
+    Worker
+  }
```

Comment:
It's purely a preference, but let's keep this on a single line:

```suggestion
  alias Oban.{Job, Queue.Executor, Repo, Worker}
```